src/generic/grid.cpp

```diff
@@ -3256,6 +3367,22 @@ wxGridCellCoordsArray wxGrid::CalcCellsExposed( const wxRegion& reg ) const
     return cellsExposed;
 }
 
+void wxGrid::PrepareDC(wxDC &dc, wxGridWindow *gridWindow)
+{
+    wxScrolledWindow::PrepareDC( dc );
+    
+    if( gridWindow )
```

Comment:
Ok, and these space changes should be done in a separate commit, gotcha!

Reply:
No, just please combine the commit fixing the style with your existing commit -- it won't be a problem in this case, as this will just add new code in correct style, so everything will be fine.

The problem with style or whitespace changes in the same commit as other, significant, changes is that the former distract from the latter (and sometimes you can glance over something important thinking that it was just a style change). But, again, if you just fix the style of the code you're adding anyhow, this problem doesn't arise.